### PR TITLE
Add transfer modal, refactor asset queries

### PR DESCRIPTION
**Pull request by dqbd**

### CODE REVIEW

File: src/api/assetDetails.ts [hunk 1]
```diff
@@ -1,54 +1,70 @@
...
-  return useQueries({
-    queries:
-      ids?.map((id) => ({
-        queryKey: QUERY_KEYS.assetDetails(id),
-        queryFn: !!id ? getAssetDetails(api, id) : undefinedNoop,
-        enabled: !!id,
-      })) ?? [],
+  const normalizedIds = ids
+    ?.filter((x): x is u32 | string => x != null)
+    .map((i) => i?.toString())
```
Comment (vkulinich-cl) [inline src/api/assetDetails.ts:22]: Ja bych radsi pouzil `reduce` funkci misto kombinace `filter` + `map` z hlediska lepsiho performance

File: src/sections/pools/pool/details/PoolDetails.utils.ts [hunk 2]
```diff
@@ -44,11 +44,11 @@ export function usePoolDetailsTradeVolume(poolAddress: string) {
     if (volume.isLoading) return null
 
     const combinedAssets = spotPrices.map((spotPrice, idx) => {
-      const asset = assets[idx]
-      if (asset.data?.data == null || spotPrice.data == null) return null
+      const asset = assets.data?.[idx]
+      if (asset == null || spotPrice.data == null) return null
```
Comment (vkulinich-cl) [inline src/sections/pools/pool/details/PoolDetails.utils.ts:48]: Myslim si, ze asset take by mohl byt `undefined`, ze?

File: src/sections/wallet/transfer/WalletTransferAccountInput.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { AccountAvatar } from "components/AccountAvatar/AccountAvatar"
+import { AddressInput } from "components/AddressInput/AddressInput"
+import { Maybe } from "utils/helpers"
+import { decodeAddress, encodeAddress } from "@polkadot/util-crypto"
+import { ReactComponent as GuestIcon } from "assets/icons/GuestIcon.svg"
+
+export const WalletTransferAccountInput = (props: {
```
Comment (vkulinich-cl) [inline src/sections/wallet/transfer/WalletTransferAccountInput.tsx:7]: Mohl bys prosim vytvorit `type` pro `WalletTransferAccountInput`

File: src/sections/pools/pool/details/PoolDetails.utils.ts (hunk 2 above)
Comment (dqbd) [reply src/sections/pools/pool/details/PoolDetails.utils.ts:48]: `asset == null` pokrývá `null` i `undefined`. 

https://262.ecma-international.org/5.1/#sec-11.9.3

**Review by vkulinich-cl — APPROVED** [review]

**Review by obsivac-cleevio — CHANGES_REQUESTED** [review]
jen maly nitpick, jinak merge

**Pull request merged**